corehq/apps/userreports/dbaccessors.py

```diff
@@ -0,0 +1,10 @@
+def get_report_configs_by_data_source(domain, data_source_id):
```

Comment:
not blocking, but just realized we have two conventions for this.

one is `userreports.dbaccessors` and the other is `userreports.dbaccessors.report_configs_by_data_source` (.py)

I think both are net better, though I favor the second one to the first. merging anyways since we're already inconsistent with this though.